### DISCUSSION TITLE
I have some problem with 389 Directory Server container project

### DISCUSSION THREAD

**Discussion by mujianfenghe** [389ds/389-ds-base#6770] [Q&A]
Hello
   I want to ask a question with using 389ds/dirsrv 389 Directory Server Container in dockerhub
when I use 389ds,the default login administrator is cn=Directory Manager,but I want to change or add another login username,How I shoud do?please help me!
Thank you very much

**Comment by vashirov** [comment]
You can use any LDAP client to connect to your server and create entries there.
If you're just starting with the DS container, it doesn't have any entries created.
You can run something like this first:
(assuming you have your container up and running using podman)
```
$ podman exec -til dsconf localhost backend create --suffix="dc=example,dc=com" --be-name userRoot --create-suffix --create-entries

The database was sucessfully created

$ podman exec -til dsidm localhost --basedn dc=example,dc=com user create --uid ldap_user --cn ldap_user --displayName "Ldap User" --uidNumber 1001 --gidNumber 1001 --homeDirectory /home/ldap_user

Successfully created ldap_user

$ podman exec -til dsidm localhost --basedn dc=example,dc=com account reset_password uid=ldap_user,ou=people,dc=example,dc=com Secret123

reset password for uid=ldap_user,ou=people,dc=example,dc=com
```
